src/main/java/com/blackduck/integration/detect/lifecycle/run/step/CommonScanStepRunner.java

```diff
@@ -86,12 +101,18 @@ public UUID performCommonUpload(NameVersion projectNameVersion, BlackDuckRunData
                 // If we can't access the SCASS uplaod URL, we create a new scanId so we can try the BDBA flow.
                 // Note: as of 2025.1.1 there is no endpoint to cancel a SCASS scan.
                 logger.info("Error uploading to SCASS URL: " + e.getMessage());
-                scanId = createFallbackScanId(operationRunner, scanType, projectNameVersion, codeLocationName, scanFile.get().length(), blackDuckRunData);
+                if (!scanType.equals(PACKAGE_MANAGER)) {
+                    scanId = createFallbackScanId(operationRunner, scanType, projectNameVersion, codeLocationName, scanFile.get().length(), blackDuckRunData);
+                }
             }
         }
-        // This is a SCASS capable server server but SCASS is not enabled or the GCP URL is inaccessible.
-        BdbaScanStepRunner bdbaScanStepRunner = createBdbaScanStepRunner(operationRunner);
-        bdbaScanStepRunner.runBdbaScan(projectNameVersion, blackDuckRunData, scanFile, scanId.toString(), scanType);
+        // This is a SCASS capable server server but SCASS is not enabled or the GCP URL is inaccessible. If PACKAGE_MANGER scan, we use the same scanId
+        if(!scanType.equals(PACKAGE_MANAGER)) {
+            BdbaScanStepRunner bdbaScanStepRunner = createBdbaScanStepRunner(operationRunner);
+            bdbaScanStepRunner.runBdbaScan(projectNameVersion, blackDuckRunData, scanFile, scanId.toString(), scanType);
+        } else {
+            isSCASSPossible = false;
```

Comment:
can we maybe add something about this being a package manager SCASS scan to the variable? I think it's only for PM scans. Maybe isPackageManagerScassPossible?